Makefile

```diff
@@ -3,8 +3,8 @@
 all: help
 
 .PHONY: install_test
-install_test: ## Install test dependencies
-	@poetry install --with=development,testing
+install_test:
```

Comment:
Please, add a comment for automatic generation of the help message like for other commands.